src/domains/wallet-app/api/payment-wallet/v1/_get_wallets_by_user_and_walletId_with_pm.xml.tpl

```diff
@@ -150,10 +150,10 @@
                         var convertedApplications = new List<JObject>();
                         foreach(JValue application in wallet["enableableFunctions"]){
                             string applicationName = application.ToString().ToUpper();
-                            if(walletApplications.Contains(applicationName)){
+                            if(walletApplications.Contains(applicationName) && wallet[application.ToString()] != null){
```

Comment:
```suggestion
                            if(walletApplications.Contains(applicationName) && wallet[applicationName] != null){
```
Why not use the `applicationName` here?

Reply:
Because `applicationName` is uppercase, while we are searching for lowercase key